bench/workgen/workgen.i

```diff
@@ -114,6 +114,7 @@
 %enddef
 
 InterruptableFunction(workgen::execute)
+InterruptableFunction(workgen::Workload::create_table)
```

Comment:
@ddanderson, is that the right thing to do, to define it as a `InterruptableFunction`?

Reply:
I don't think so. it's a very quick function.  Also the create_table function will generally be called while workload.run() is running, and that has already disabled signal handlers.  See my continued comment in the main conversation.

Reply:
Thanks, removed in 7daa57da6b6f5d0f976d05700d0beb633c163485